index.bs

```diff
@@ -79,30 +72,14 @@ beyond those described in the Generic Sensor API [[!GENERIC-SENSOR]].
 Model {#model}
 =====
 
-The Proximity Sensor's associated <a>Sensor subclass</a>
-is the {{ProximitySensor}} class.
-
-The Proximity Sensor's associated <a>SensorReading subclass</a>
-is the {{ProximitySensorReading}} class.
+The proximity sensor's associated {{Sensor}} subclass is the {{ProximitySensor}} class.
 
 The Proximity Sensor has a <a>default sensor</a>,
 which is the device's main proximity detector.
 
-The Proximity Sensor has a single <a>supported reporting mode</a>
-which is "<a>auto</a>".
+A [=latest reading=] per [=sensor=] of proximity sensor type includes three [=map/entries=]
+whose [=map/keys=] are "distance", "max", "near" and whose [=map/values=] contain {{ProximitySensor/distance!!attribute}}, {{ProximitySensor/max!!attribute}} and {{ProximitySensor/near!!attribute}} values.
```

Comment:
I believe here it rather should be an actual value description (i.e. distance between the sensor and the closest visible surface) rather than just {{ProximitySensor/distance!!attribute}} as [=sensor=] here means an actual sensor, not a ProximitySensor object.
Now there is a circular dependency in the declaration: latest reading contains value of an attribute and the attribute contains the latest reading field.